thunder/executors/cudnnex.py

```diff
@@ -367,6 +387,31 @@ def _cudnn_sdpa_checker(
         if d % 8 != 0 or d > 128:
             return False
 
+    try:
+        # Build both forward and backward graphs
+        query_4d, key_4d, value_4d, attn_mask_4d = _transform_sdpa_inputs(query, key, value, attn_mask)
+        _make_cudnn_sdpa_forward_graph(query_4d, key_4d, value_4d, attn_mask_4d, dropout_p, is_causal)
+        _make_cudnn_sdpa_backward_graph(
+            query_4d,
+            key_4d,
+            value_4d,
+            attn_mask_4d,
+            dropout_p,
+            is_causal,
+            query_4d.stride,
+            key_4d.stride,
+            value_4d.stride,
+        )
+
+    # If cudnn can't support the graph, return false
+    # Please turn on cudnn API logging for helpful messages that mention why the graph is not supported.
```

Comment:
Should thunder turn this on itself anytime graph creation fails? @mruberry or @IvanYashchuk thoughts?

Reply:
I think not by default. The default path shouldn't have warnings on every call. But it would be useful to have a flag that turns on this behavior.